packages/node-services/src/child_process.ts

```diff
@@ -34,15 +34,15 @@ class Stream extends EventEmitter {
     super();
   }
 
-  setEncoding(encoding: string) {}
+  setEncoding() {}
 
-  write(message: string, encoding: string) {
+  write(message: string) {
```

Comment:
Are we required to remove these parameters? I think they have a documentation purpose. If linting requires it it's okay to remove them.

Reply:
They trigger the `no-undefined` rule